homeassistant/components/ezviz/camera.py

```diff
@@ -263,6 +263,16 @@ async def stream_source(self) -> str | None:
 
     def perform_ptz(self, direction: str, speed: int) -> None:
         """Perform a PTZ action on the camera."""
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "service_depreciation_ptz",
+            breaks_in_ha_version="2023.9.0",
```

Comment:
A bit related to this ADR: <https://github.com/home-assistant/architecture/blob/master/adr/0021-YAML-integration-configuration-deprecation-policy.md>

Should be make this longer?